Js/popup.js

```diff
@@ -0,0 +1,66 @@
+import popupdata from './data.js';
+
+const submit = document.querySelectorAll('.see-project');
+const popopcontainer = document.querySelector('.popup-container');
+const popupwindow = (data) => {
+  popopcontainer.innerHTML = ` <div class="container-popup">
+   <div class="content">
+   <div class="no idea">
+       <div class="header-cross">
+     <h2 class="content-header margin-none">${data.projectname}</h2>
+     <p class="cross-popup"> <img class="cross" src="./images/cross.png" alt=""></p>
+   </div>
+   <div class="pop-header-tech">
+     <ul class="specification margin-none margin-bottom">
+         <li class="name">${data.frame[0]}</li>
+         <li class="skills"><img src="${data.dot}" alt="">
+         <p>${data.frame[1]}</p></li> 
+         <li class="year"><img src="${data.dot}" alt="">
+           <p>${data.frame[2]}</p></li>
+     </ul>
+   </div>
+   </div>
+     <div class="image">
+       <img class="img pop-image" src="${data.popimage}" alt="">
```

Comment:
- [ ] Kindly fix the image overstretch on all your popup cards for the desktop view. You can fix this by removing the fixed height you added for the images.
your image 👇🏽 
![image](https://github.com/alizaamir123/mobile-version/assets/63473584/d2bb13af-46fc-4ba2-8228-45308b60d354)
 
figma image 👇🏽 
![image](https://github.com/alizaamir123/mobile-version/assets/63473584/8b1cea15-6867-4bf1-ae3a-5814b876cca0)